app/views/tracks/about.html.haml

```diff
@@ -1,4 +1,4 @@
-- track_meta_tags(@track)
+- track_meta_tags(@user_track)
```

Comment:
No, because then `@user_track` will be `UserTrack::External`. See https://github.com/exercism/website/blob/main/app/controllers/tracks_controller.rb#L51 for  the call to create the user track, and https://github.com/exercism/website/blob/0f0a8ba30660cdfcf17b66e9bb6060821e3ceeb8/app/models/user_track.rb#L38 for its implementation. The user track will only ever be `nil` if the track can't be found, but that is already covered in https://github.com/exercism/website/blob/main/app/controllers/tracks_controller.rb#L50

Reply:
Oh, and we're already using the user track to display the exercises on that page: https://github.com/exercism/website/blob/main/app/views/tracks/about.html.haml#L17

Reply:
Ah nice!